lib/views/BinnedMeanGraph.js

```diff
@@ -105,16 +107,16 @@ class BinnedMeanGraph extends Graph {
                 .attr('class', 'binned_mean_guide binned_mean_x_guide');
 
             super.drawXLineGuide(xGuide, xG, chartOpts, {
-                xProp: 'x_mean',
-                yProp: 'y_mean',
+                xProp          : 'x_mean',
+                yProp          : 'y_mean',
```

Comment:
This particular one might be Webstorm's opinion rather than mine 😄 - I just used the auto-indent function on this piece. But whatever, I don't mind this formatting either, so in it goes!